sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_polling.py

```diff
@@ -101,11 +113,13 @@ def details(self) -> Mapping[str, Any]:
             created_on = datetime.datetime.strptime(created_on, "%Y-%m-%dT%H:%M:%SZ")
         last_updated_on = self._current_body.get("lastUpdatedDateTime", None)
         if last_updated_on:
-            last_updated_on = datetime.datetime.strptime(last_updated_on, "%Y-%m-%dT%H:%M:%SZ")
+            last_updated_on = datetime.datetime.strptime(
```

Comment:
I'm surprised that shorter lines like this got reformatted -- did you run `black` with `-l 120`?

Reply:
Hmmm I did not let me check if that changes it.

Reply:
For context, `black` defaults to enforcing a max line length of 90 characters, but we usually enforce a max of 120 in the SDK. Ultimately shorter is better than longer, but worth double-checking 🙂

Reply:
Ok ran with -l 120 and it fixed spacing due to line length that was applied during several of my previous black runs on this code. I shall try to remember to always run it with `-l 120`. Thanks McCoy!